xorgauth/urls.py

```diff
@@ -23,6 +23,13 @@
 from xorgauth.relying_party_test import views as rptest_views
 from xorgauth import forms as xorgauth_forms
 
+
+if hasattr(auth_views, 'PasswordChangeDoneView'):
+    # Django 2.1 replaced auth_password_change_done_view by PasswordChangeDoneView
+    auth_password_change_done_view = auth_views.PasswordChangeDoneView.as_view()
+else:
+    auth_password_change_done_view = auth_views.password_change_done
```

Comment:
Honest question: do we want to fill the codebase with such tests as every new version of django is released, rather than just update & bump our minimal required version?

Reply:
As we are still using Django 1.11 (because of Python 2.7), yes for now.

Reply:
Oh right, forgot about that sorry.